salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -1,12 +1,19 @@
+{% from "orchestrate/aws_env_macro.jinja" import VPC_NAME, VPC_RESOURCE_SUFFIX,
+ ENVIRONMENT, BUSINESS_UNIT, PURPOSE_PREFIX, subnet_ids with context %}
+
 {% set profile_overrides = profile_overrides|default({}) %}
 {% set tags = tags|default({}) %}
-{% set release_number = salt.sdb.get('sdb://consul/edxapp-release-version') %}
+{% set env_settings = salt.pillar.get('environments:({})'.format(ENVIRONMENT)) %}
+{% set purposes = env_settings.purposes %}
+{% set codename = purposes[PURPOSE_PREFIX +'-live'].versions.codename %}
+{% set release_number = salt.sdb.get('sdb://consul/edxapp-{}-release-version').format(codename) %}
```

Comment:
The `.format` call needs to be moved inside the `)` so it gets applied to the sdb URL. As it is right now it will get applied to the return value of the sdb call.